doc/input-output-reference/src/input-for-output.tex

```diff
@@ -653,15 +653,16 @@ \subsection{Output:EnvironmentalImpactFactors}\label{outputenvironmentalimpactfa
 
 \begin{longtable}[c]{@{}ll@{}}
 \toprule
+%Electricity:Facility & Diesel:Facility \tabularnewline
+%\midrule
+%\endfirsthead
+%
+%\toprule
+%Electricity:Facility & Diesel:Facility \tabularnewline
+%\midrule
+%\endhead
+%
```

Comment:
Why?

Reply:
The basic idea here is that for this table only a top rule is needed and then the two columns. 
However, it is indeed a little perplex in the original form why it need two sets: one if from Line 655 to Line 658, ending with `endfirsthead`; and the second set from Line 660 to Line 663, ending with `endhead`---but everything else is the same. Maybe some specification in the `longtable`, but here I jsut need the outer frame (so only top rule) would be good--so a simplification is made here to keep only two lines.